### PR TITLE
Simplify starting of jenkins jobs

### PR DESCRIPTION
**Pull request by locriandev**

### CODE REVIEW

File: pyartcd/pyartcd/jenkins.py [hunk 1]
```diff
@@ -185,15 +186,15 @@ def start_ocp4(build_version: str, assembly: str, rpm_list: list,
         params['BUILD_IMAGES'] = 'none'
 
     return start_build(
-        job_name=Jobs.OCP4.value,
+        job_name=Jobs.OCP4,
```
Comment (thegreyd) [inline pyartcd/pyartcd/jenkins.py:189]: Think these params would need change `job_name -> job`

Comment (locriandev) [reply pyartcd/pyartcd/jenkins.py]: Right, fixed

**Pull request merged**